internal/cmd/users.go

```diff
@@ -115,22 +118,34 @@ func newUsersListCmd(cli *CLI) *cobra.Command {
 				return err
 			}
 
-			for _, user := range users.Items {
-				rows = append(rows, row{
-					Name:       user.Name,
-					LastSeenAt: user.LastSeenAt.Relative("never"),
-				})
-			}
+			switch format {
+			case "text":
```

Comment:
Since this is the default, it doesn't need it a name. Let's move this to `default` below